R/serialize.R

```diff
@@ -66,6 +68,28 @@ serialize_parse_data_flat <- function(pd_flat) {
         collapse = "")) %>%
     .[["text_ws"]] %>%
     strsplit("\n", fixed = TRUE) %>%
-    .[[1L]]
+    .[[1L]] %>%
+    trimws(which = "right")
```

Comment:
We did the trimming in the initial text exclusively for the roundtrip test, which we currently don't have. For serialization, we're inserting spaces ourselves, and I'd rather avoid inserting extra space (by adapting `newlines_and_spaces()` if necessary) instead of trimming after the fact.